akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -200,6 +200,8 @@ on all nodes::
 You must explicitly add the ``akka-distributed-data-experimental`` dependency to your build if
 you use this mode. It is possible to remove ``akka-persistence`` dependency from a project if it
 is not used in user code and ``remember-entities`` is ``off``.
+Using it together with ``Remember Entities`` shards will be recreated after rebalancing, however will
```

Comment:
Shouldn't this document explicitly that persistence is used for storing the remembered entries?